src/App.jsx

```diff
@@ -74,13 +74,13 @@ class App extends Component {
 
   constructor(props) {
     super(props);
-    var allProjects = [...data.Projs];
-    allProjects.forEach(function(project, index) {
+    var projects = [...data.Projs];
+    projects.forEach(function(project, index) {
```

Comment:
Prefer arrow functions, otherwise you won't be able to access `this` from the class without a re-bind.

Reply:
Noted. changed in other PR! Thx!!:D